src/igvfd/upgrade/sample.py

```diff
@@ -192,10 +192,10 @@ def whole_organism_10_11(value, system):
 def whole_organism_11_12(value, system):
```

Comment:
I am not 100% sure this upgrade change is "kosher" or even going to work, as in schema 11 there was no sample_terms there?

Reply:
I think it is necessary, the upgrade fails if I don't make this change (I think the issue is that even though v12 didn't exist at the time, there isn't a stored version WholeOrganism schema_version:11 to run the upgrade with)
>       if value['biosample_term'] != ['/sample-terms/UBERON_0000468/']:
E       KeyError: 'biosample_term'